src/cljs/pyregence/state.cljs

```diff
@@ -119,6 +117,8 @@ California cameras. This atom is used to create the camera layer in mapbox.cljs.
   md-available-dates (r/atom {}))
 (defonce ^{:doc "An integer that keeps track of the number of active fires."}
  active-fire-count (r/atom 0))
+(defonce timezone ^{:doc "The current selected timezone"}
```

Comment:
Let's change this to `*timezone` to reflect that it's holding the state of something currently selected in the UI. We've arbitrarily decided to use an asterisk preceding a piece of state for state such as this one (e.g. `*forecast`).